Index: src/test/java/tests/PageObjectLoginTest.java
===================================================================
--- src/test/java/tests/PageObjectLoginTest.java	(revision 1866a443eaedce905277571866e6f524dfffe4dc)
+++ src/test/java/tests/PageObjectLoginTest.java	(revision 1866a443eaedce905277571866e6f524dfffe4dc)
@@ -1,37 +0,0 @@
-package tests;
-
-import org.junit.Before;
-import org.junit.Test;
-import pages.LoginPage;
-
-public class PageObjectLoginTest extends BaseTest{
-
-    private LoginPage loginPage;
-
-    @Before
-    public void init(){
-        this.loginPage = new LoginPage(this.driver);
-    }
-
-    @Test
-    public void negativeAuthTest(){
-        this.loginPage.login("TestUser", "TestPassword")
-                .validateError("Incorrect username or password.");
-    }
-
-    @Test
-    public void positiveAuthTest(){
-        this.loginPage.login()
-                .validateSuccess("Learn Git and GitHub without any code!");
-    }
-
-
-    @Test
-    public void checkOurCommits(){
-        this.loginPage.login()
-                .openCommits();
-    }
-
-
-
-}
Index: src/test/java/Lesson/tests/PageObjectLoginTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/Lesson/tests/PageObjectLoginTest.java	(date 1597567287027)
+++ src/test/java/Lesson/tests/PageObjectLoginTest.java	(date 1597567287027)
@@ -0,0 +1,37 @@
+package Lesson.tests;
+
+import org.junit.Before;
+import org.junit.Test;
+import Lesson.pages.LoginPage;
+
+public class PageObjectLoginTest extends tests.BaseTest {
+
+    private LoginPage loginPage;
+
+    @Before
+    public void init(){
+        this.loginPage = new LoginPage(this.driver);
+    }
+
+    @Test
+    public void negativeAuthTest(){
+        this.loginPage.login("TestUser","TestPassword")
+                .validateError("Incorrect username or password.");
+    }
+
+    @Test
+    public void positiveAuthTest(){
+        this.loginPage.login()
+                .validateSuccess("Working with a team?");
+    }
+
+
+    @Test
+    public void checkOurCommits(){
+        this.loginPage.login()
+                .openCommits();
+    }
+
+
+
+}
Index: src/test/java/tests/BaseTest.java
===================================================================
--- src/test/java/tests/BaseTest.java	(revision 1866a443eaedce905277571866e6f524dfffe4dc)
+++ src/test/java/Lesson/tests/BaseTest.java	(date 1597524220631)
@@ -16,7 +16,7 @@
 
 public abstract class BaseTest {
 
-    protected final Logger log = LogManager.getLogger("Test");
+    private final Logger log = LogManager.getLogger("Test");
 
     protected WebDriver driver;
 
@@ -24,7 +24,7 @@
     public void setUp(){
         System.setProperty("webdriver.chrome.driver",
                 System.getProperty("user.dir") +
-                        "/src/main/resources/drivers/chrome/v84/chromedriver");
+                        "/src/main/resources/drivers/chrome/v84/chromedriver.exe");
         switch (System.getProperty("browser", "chrome")){
             case "chrome":
                 this.driver = new ChromeDriver();
Index: src/test/java/tests/FactoryTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/tests/FactoryTest.java	(revision 1866a443eaedce905277571866e6f524dfffe4dc)
+++ src/test/java/tests/FactoryTest.java	(date 1597524086889)
@@ -2,11 +2,9 @@
 
 import org.junit.Before;
 import org.junit.Test;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.support.FindBy;
-import pages.PageFactory;
+import Lesson.pages.PageFactory;
 
-public class FactoryTest extends BaseTest{
+public class FactoryTest extends tests.BaseTest {
 
     private PageFactory factory;
 
Index: src/test/java/pages/MainPage.java
===================================================================
--- src/test/java/pages/MainPage.java	(revision 1866a443eaedce905277571866e6f524dfffe4dc)
+++ src/test/java/Lesson/pages/MainPage.java	(date 1597567203948)
@@ -1,4 +1,4 @@
-package pages;
+package Lesson.pages;
 
 import org.junit.Assert;
 import org.openqa.selenium.By;
@@ -11,7 +11,7 @@
 
     }
 
-    private final By successHead = By.xpath("//h2[@class = 'shelf-title']");
+    private final By successHead = By.xpath("//h2[@class='hide-sm hide-md f5 mb-1 border-top pt-3']");
     private final String successMessage = "Learn Git and GitHub without any code!";
 
     private final By commitButton = By.xpath("//span[@class = 'css-truncate css-truncate-target']");
Index: src/test/java/pages/PageFactory.java
===================================================================
--- src/test/java/pages/PageFactory.java	(revision 1866a443eaedce905277571866e6f524dfffe4dc)
+++ src/test/java/Lesson/pages/PageFactory.java	(date 1597524548702)
@@ -1,4 +1,4 @@
-package pages;
+package Lesson.pages;
 
 import org.junit.Assert;
 import org.openqa.selenium.WebDriver;
@@ -19,7 +19,7 @@
     @FindBy(xpath = "//div[@class = 'container-lg px-2']")
     private WebElement errorMessage;
 
-    @FindBy(xpath = "//h2[@class = 'shelf-title']")
+    @FindBy(xpath = "//h2[@class='hide-sm hide-md f5 mb-1 border-top pt-3']")
     private WebElement successHead;
 
     private WebDriver driver;
Index: src/test/java/pages/BasePage.java
===================================================================
--- src/test/java/pages/BasePage.java	(revision 1866a443eaedce905277571866e6f524dfffe4dc)
+++ src/test/java/Lesson/pages/BasePage.java	(date 1597517242044)
@@ -1,4 +1,4 @@
-package pages;
+package Lesson.pages;
 
 import org.openqa.selenium.WebDriver;
 
Index: src/test/java/pages/LoginPage.java
===================================================================
--- src/test/java/pages/LoginPage.java	(revision 1866a443eaedce905277571866e6f524dfffe4dc)
+++ src/test/java/Lesson/pages/LoginPage.java	(date 1597567085993)
@@ -1,4 +1,4 @@
-package pages;
+package Lesson.pages;
 
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
Index: src/test/java/DZ12/BaseTest.java
===================================================================
--- src/test/java/DZ12/BaseTest.java	(revision 1866a443eaedce905277571866e6f524dfffe4dc)
+++ src/test/java/DZ12/Tests/BaseTest.java	(date 1597608763055)
@@ -1,4 +1,4 @@
-package DZ12;
+package DZ12.Tests;
 
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
Index: src/test/java/DZ12/GitTest.java
===================================================================
--- src/test/java/DZ12/GitTest.java	(revision 1866a443eaedce905277571866e6f524dfffe4dc)
+++ src/test/java/DZ12/Tests/GitTest.java	(date 1597608763029)
@@ -1,6 +1,7 @@
-package DZ12;
+package DZ12.Tests;
 
 import DZ12.Pages.LoginPage;
+import DZ12.Tests.BaseTest;
 import org.junit.Before;
 import org.junit.Test;
 
Index: src/test/java/Lesson/tests/FactoryTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/Lesson/tests/FactoryTest.java	(date 1597525736648)
+++ src/test/java/Lesson/tests/FactoryTest.java	(date 1597525736648)
@@ -0,0 +1,24 @@
+package Lesson.tests;
+
+import Lesson.pages.PageFactory;
+import org.junit.Before;
+import org.junit.Test;
+import tests.BaseTest;
+
+
+public class FactoryTest extends BaseTest {
+
+    private PageFactory factory;
+
+    @Before
+    public void init(){
+        factory = new PageFactory(driver);
+
+    }
+
+    @Test
+    public void checkLogin(){
+       factory.login();
+    }
+
+}
